src/components/product-filters/product-filters.hbs

```diff
@@ -89,14 +89,23 @@
             <form data-js="price-filter" class="price-filter__form accordion__panel filter__options">
                 <input type="hidden" name="priceFilterFrom" id="priceFilterFrom" value="">
                 <input type="hidden" name="priceFilterTo" id="priceFilterTo" value="">
+                <p class="error-summary hidden" id="errorSummaryPriceFilters" role="alert">You have <span></span> in your price range.</p>
                 <fieldset class="price-filter__fieldset">
                     <legend id="priceLegend" class="visibly-hidden">Filter by Price</legend>
-                    <label for="priceFrom" class="price-filter__label">
-                        <span class="sr-only">Price </span>From <input type="text" name="priceFrom" id="priceFrom" class="price-filter__input">
-                    </label>
-                    <label for="priceTo" class="price-filter__label">
-                        <span class="sr-only">Price </span>To <input type="text" name="priceTo" id="priceTo" class="price-filter__input">
-                    </label>
+                    <div class="price-filter__field-container">
```

Comment:
I had to add a wrapper element so that the error message appears below the input field, instead of beside it.